### ISSUE TITLE
 Cannot read property 'lineComments' of undefined   Code: -32603

### ISSUE THREAD

**Issue by opsarno** [redhat-developer/vscode-yaml#358]
```
[Error - 6:24:54 PM] Request textDocument/documentSymbol failed.
  Message: Request textDocument/documentSymbol failed with message: Cannot read property 'lineComments' of undefined
  Code: -32603
```

This problem occurs when I cut back YAML files from other files for editing.

Exp:
```yaml
# test comment
- name: xxxxx
```

Same as https://github.com/redhat-developer/vscode-yaml/issues/353

**Comment by huytn** [comment]
and
```
Error - 3:29:10 PM] Request textDocument/hover failed.
  Message: Request textDocument/hover failed with message: Cannot read property 'lineComments' of undefined
  Code: -32603 
```

**Comment by alexkoepke** [comment]
Getting the same issue error when hovering over comments.
```
[Error - 3:55:15 PM] Request textDocument/hover failed.
  Message: Request textDocument/hover failed with message: Cannot read property 'lineComments' of undefined
  Code: -32603 
```

**Comment by namoshizun** [comment]
Having encountered this issue as well after upgrading VSCode to v1.48.2 on  Debian.  

```
[Error - 10:22:57 AM] Request textDocument/documentSymbol failed.
  Message: Request textDocument/documentSymbol failed with message: Cannot read property 'lineComments' of undefined
  Code: -32603 
```

**Comment by gchristidis** [comment]
I am working on AWS CF yaml docs and i get this message every second and i cant find a way to turn it off, closing the output panel just opens it up again when the next message is displayed.  Its very annoying

```
[Error - 11:00:51 am] Request textDocument/hover failed.
  Message: Request textDocument/hover failed with message: Cannot read property 'lineComments' of undefined
  Code: -32603 
```

**Comment by JPinkney** [comment]
This should be fixed in the 0.10.1 release that just went out
